scripts/common

```diff
@@ -86,6 +86,12 @@ else
   )
 fi
 
+if [[ "${OSTYPE}" == "darwin"*  ]]; then
+  export RUST_HOST_TARGET="${RUST_HOST_TARGET:-x86_64-apple-darwin}"
```

Comment:
Why do these need to be exported? I thought just declaring them (and in fact as `readonly`) should be enough. I don't trust myself with bash scripts though, so I'm probably wrong.

Reply:
I think `RUST_HOST_TARGET` is an environment variable that might be set beforehand.

Reply:
I'm not sure I follow, who should set this env variable then? Or is this just a convoluted way to avoid specifying a proper flag to the relevant bash scripts?

Reply:
I *think* we'd be OK with:
```bash
readonly RUST_HOST_TARGET="${RUST_HOST_TARGET:-x86_64-apple-darwin}"
```

(That would leave it as just a shell variable if it's not already an environment variable, but I don't think we need an environment variable &ndash; it's only used in scripts that `source` this one, at which point a shell variable is fine.)

Reply:
Why should this one be different from `RUST_FLAG`, which is set using export in line 11 of the same file? 
https://github.com/project-oak/oak/blob/56ee7fa6b625b59244eff9b45467af5ec662613a/scripts/common#L11

My understanding from your original comment was that you wanted it to be an environment variable so that people have the flexibility to set it in their environment to whatever target they want.

Reply:
You are right, `RUST_LOG` has the same issue (it is an extra way of passing parameters to the scripts, in addition to the explicit script flags).

I was expecting that people would pass something explicit if they want a non-default behaviour (especially if the default behaviour is sensible, such as in this case, with the extra logic around it).

Also env variables seem hard / impossible to discover (while hopefully flags show up with `-h`, even though it is very manual in bash).

But I guess some people may prefer to pass it as variable too.

Anyways, just something to think about, if we start piling up on this behaviour we should think how to document it (it may just be that we refer to the implementation of `scripts/common`, at a minimum).

Reply:
BTW, `RUST_LOG` is a bit different, in that it has to be an environment variable or it won't work (it gets passed via the environment to a separately executed process).

Reply:
In that case, just a simple assignment (`readonly RUST_HOST_TARGET="x86_64-apple-darwin"`) is perhaps better. We can make that changed later if that makes sense.